### PR TITLE
Update Parachain commitments worker to watch and relay from beefy

### PR DESCRIPTION
**Pull request by musnit**
This PR removes the old parachaincommitment worker's listener that was watching the parachain and now depends only on the beefy listener, reacting to beefy messages.

e2e tests all pass

Still to do in follow up PRs:
 - Rework catchup mechanism so that it works with beefy watching/beefy proofs
 - Update beefy-listener to watch the smart contract light client rather than the actual relaychain
 - Send through real proof data and get it working with real smart contracts once checks are enabled in @philipstanislaus's next PR

### CODE REVIEW

File: relayer/workers/parachaincommitmentrelayer/beefy-listener.go [hunk 1]
```diff
@@ -28,11 +26,11 @@ const OUR_PARACHAIN_ID = 200
...
 
 type MessagePackage struct {
-	channelID          chainTypes.ChannelID
-	commitmentHash     types.H256
-	commitmentMessages []chainTypes.CommitmentMessage
-	paraHeadProof      string
-	mmrProof           types.GenerateMMRProofResponse
+	channelID              chainTypes.ChannelID
+	commitmentHash         types.H256
+	commitmentMessagesData types.StorageDataRaw
```
Comment (vgeddes) [inline relayer/workers/parachaincommitmentrelayer/beefy-listener.go:31]: I see you've finally come over to mine and Beefy's choice of naming, in calling the hash of the commitment the `commitmentHash`. 😆 

Nit: Can you shorten `commitmentMessagesData`? Its a bit verbose.
```suggestion
	commitmentData types.StorageDataRaw
```

File: relayer/workers/parachaincommitmentrelayer/beefy-listener.go [hunk 2]
```diff
@@ -278,56 +265,61 @@ func createParachainHeaderProof(allParaHeads []types.Header, ourParaHead types.H
...
-			return &auxDigestItem, nil
+			auxDigestItems = append(auxDigestItems, auxDigestItem)
 		}
 	}
-	return nil, nil
+	return auxDigestItems, nil
 }
 
-func (li *BeefyListener) getMessagesForDigestItem(digestItem *chainTypes.AuxiliaryDigestItem) ([]chainTypes.CommitmentMessage, error) {
+func (li *BeefyListener) getMessagesDataForDigestItem(digestItem *chainTypes.AuxiliaryDigestItem) (types.StorageDataRaw, error) {
```
Comment (vgeddes) [inline relayer/workers/parachaincommitmentrelayer/beefy-listener.go:322]: Nit: There's no concept of a message in this scope.

```suggestion
func (li *BeefyListener) getDataForDigestItem(digestItem *chainTypes.AuxiliaryDigestItem) (types.StorageDataRaw, error) {
```

**Review by vgeddes — COMMENTED** [review]
Looks good! Some minor things like naming can be improved though.

File: relayer/workers/parachaincommitmentrelayer/beefy-listener.go (hunk 2 above)
Comment (musnit) [reply relayer/workers/parachaincommitmentrelayer/beefy-listener.go:322]: will do

File: relayer/workers/parachaincommitmentrelayer/beefy-listener.go (hunk 1 above)
Comment (musnit) [reply relayer/workers/parachaincommitmentrelayer/beefy-listener.go:31]: will do

**Comment by musnit** [comment]
merging - will address extra tasks in new PRs

**Pull request merged**